text/000x-assethub.md

```diff
@@ -0,0 +1,92 @@
+# RFC-0000: Lowering NFT Deposits on Polkadot and Kusama Asset Hubs
+
+|                 |                                                                                                    |
+| --------------- | -------------------------------------------------------------------------------------------------- |
+| **Start Date**  | 2 November 2023                                                                                    |
+| **Description** | A proposal to reduce the minimum deposit required for collection creation on the Polkadot and Kusama Asset Hub, making it more accessible and affordable for artists. |
+| **Authors**     | [Aurora Poppyseed](https://github.com/poppyseedDev), [Just_Luuuu](https://github.com/justLuuuu), [VikiiVal](https://github.com/vikiival)  |
+
+## Summary
+
+This RFC proposes a change to the current deposit requirements on the Polkadot and Kusama Asset Hub for creating NFT collections. The objective is to lower the barrier to entry for artists, fostering a more inclusive and vibrant ecosystem, while maintaining network integrity and preventing spam.
+
+## Motivation
+
+The current deposit of 10 DOT for collection creation on the Polkadot Asset Hub presents a significant financial barrier for many artists. By lowering the deposit requirements, we aim to encourage more artists to participate in the Polkadot NFT ecosystem, thereby enriching the diversity and vibrancy of the community and its offerings.
+
+## Stakeholders
+
+- **Artists**: Primary beneficiaries of the proposed change, particularly those who found the current deposit requirements prohibitive.
+- **KodaDot**: As the facilitator of artists' relations, KodaDot has a vested interest in making the platform more accessible.
+- **Polkadot Community**: Stands to benefit from an influx of artists, creators and diverse NFT collections, enhancing the overall ecosystem.
+
+Previous discussions have been held within the KodaDot community, as well as with artists expressing their concerns about the deposit amounts. Referencing to [Polkadot Forum conversation](https://forum.polkadot.network/t/polkadot-assethub-high-nft-collection-deposit/4262).
+
+## Explanation
+This RFC suggests modifying the smart contract governing collection creation on the Polkadot Asset Hub to require a lower deposit. The exact amount of the reduced deposit is yet to be determined and should be discussed and agreed upon by the stakeholders. The implementation of this change requires careful consideration of the network's integrity and the prevention of spam, possibly through alternative means such as rate limiting or account verification.
+
+**Prices and Proposed Prices on Polkadot Asset Hub:**
```

Comment:
The explanation lacks discussion that [I was hoping for](https://forum.polkadot.network/t/polkadot-assethub-high-nft-collection-deposit/4262/17?u=joepetrowski) about the tradeoffs and how you arrived at these suggested deposits.